docs/guides/update_versions.md

```diff
@@ -1,26 +1,25 @@
 ---
-title: Updating versions of Rocky
+title: Performing unsupported upgrades between Major Rocky Linux releases
 author: Steven Spencer
 contributors:
 ---
 
-(OR **How to duplicate a machine install**)
 
```

Comment:
Is there a reason for removing the "OR How to duplicate a machine install"? The reason I ask is that this procedure all came out of a forum post where the user had made his boot partition too small, and wanted to know an easy way to reinstall and get everything back. In other words, going from 9 to 9.